gamemodes/terrortown/entities/weapons/weapon_wwf_claws.lua

```diff
@@ -0,0 +1,266 @@
+            else
+                util.Effect("Impact", edata)
+            end
+        end
+    else
+        self:SendWeaponAnim(ACT_VM_MISSCENTER)
+    end
+
+    if not CLIENT and IsPlayer(hitEnt) and not hitEnt:ShouldActLikeJester() then
+        local dmg = DamageInfo()
+        dmg:SetDamage(self.Primary.Damage)
+        dmg:SetAttacker(owner)
+        dmg:SetInflictor(self)
+        dmg:SetDamageForce(owner:GetAimVector() * 5)
+        dmg:SetDamagePosition(owner:GetPos())
+        dmg:SetDamageType(DMG_SLASH)
+
+        hitEnt:DispatchTraceAttack(dmg, spos + (owner:GetAimVector() * 3), sdest)
+    end
+
+    if owner.LagCompensation then
+        owner:LagCompensation(false)
+    end
+end
+
+--[[
+Jump Attack
+]]
+
+function SWEP:SecondaryAttack()
+    if not werewolf_leap_enabled:GetBool() then return end
+
+    local owner = self:GetOwner()
+    if not IsValid(owner) then return end
+
+    if not self:CanSecondaryAttack() or not owner:IsOnGround() then return end
+
+    self:SetNextSecondaryFire(CurTime() + self.Secondary.Delay)
+
+    if SERVER then
+        owner:SetVelocity(owner:GetForward() * 200 + Vector(0,0,400))
+    end
+
+    -- Make this use the leap animation
+    self.ActivityTranslate[ACT_MP_JUMP] = ACT_ZOMBIE_LEAPING
+
+    -- Make it look like the player is jumping
+    hook.Run("DoAnimationEvent", owner, PLAYERANIMEVENT_JUMP)
+
+    -- Sync this jump override to the other players so they can see it too
+    if SERVER then
+        net.Start("TTT_WerewolfLeapStart")
+        net.WritePlayer(owner)
+        net.Broadcast()
+    end
+end
+
+function SWEP:Think()
+    if self.ActivityTranslate[ACT_MP_JUMP] == nil then return end
+
+    local owner = self:GetOwner()
+    if not IsValid(owner) or owner.m_bJumping then return end
+
+    -- When the player hits the ground or lands in water, reset the animation back to normal
+    if owner:IsOnGround() or owner:WaterLevel() > 0 then
+        self.ActivityTranslate[ACT_MP_JUMP] = nil
+
+        -- Sync clearing the override to the other players as well
+        if SERVER then
+            net.Start("TTT_WerewolfLeapEnd")
+            net.WritePlayer(owner)
+            net.Broadcast()
+        end
+    end
+end
+
+function SWEP:OnDrop()
+    self:Remove()
+end
+
+function SWEP:Deploy()
+    self.Primary.Damage = werewolf_attack_damage:GetInt()
+    self.Primary.Delay = werewolf_attack_delay:GetFloat()
+
+    local vm = self:GetOwner():GetViewModel()
+    vm:SendViewModelMatchingSequence(vm:LookupSequence("fists_draw"))
+end
+
+function SWEP:Holster(weap)
+    if CLIENT and IsValid(weap) then
+        local owner = weap:GetOwner()
+        if not IsPlayer(owner) then return end
+
+        local vm = owner:GetViewModel()
+        if not IsValid(vm) or vm:GetColor() == COLOR_WHITE then return end
+
+        vm:SetColor(COLOR_WHITE)
+    end
+    return true
```

Comment:
This should probably be set to `false` so the player can't switch away from the claws (I noticed you're dropping all weapons except the crowbar so I assume they have that still?)
```suggestion
    return false
```